sections/changes.include

```diff
@@ -9,6 +9,7 @@
 
 <dl>
  <dt>Removed for lack of implementation</dt>
+  <dd><a href="https://github.com/w3c/html/pull/113"><code>form</code> attribute on <{label}> element.</dd>
```

Comment:
It is implemented, this rather belong in a "Removed because it doesn't and make sense and maybe now browsers will drop support too" section. The necessary investigation is happening in https://github.com/whatwg/html/issues/787